.circleci/config.yml

```diff
@@ -104,8 +103,41 @@ jobs:
             npm install -g npm@latest
             nvm use node
             npm install
-            npx truffle migrate 0xa508dD875f10C33C52a8abb20E16fc68E981F186 1
+            npx truffle migrate --accounts 0xa508dD875f10C33C52a8abb20E16fc68E981F186 --confirmations 1
           working_directory: ~/repo/MultiSigWallet
+      - run:
+          name: Install node 12
+          command: |
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+            nvm install v12
+            node --version
+            nvm alias default v12
+            echo 'export PATH="$(npm bin):$PATH"' >> $BASH_ENV
+      - restore_cache:
+          keys:
+            - dependencies-machine-v1-{{ checksum "~/repo/plasma_framework/package-lock.json" }}
+      - run:
+          name: Install dependencies
+          working_directory: ~/repo/plasma_framework
+          command: |
+            npm version
+            node --version
+            npm install
+      - save_cache:
+          key: dependencies-machine-v1-{{ checksum "~/repo/plasma_framework/package-lock.json" }}
+          paths:
+            - ~/repo/plasma_framework/node_modules
+      - run:
+          name: Deploy Plasma Contracts
+          command: |
+            npx truffle migrate --network local
```

Comment:
I think DevOps has been asking for a docker to deploy things. We have a Dockerfile in the repo for this exact purpose and is tested on rinkeyby for each master merging (see the job `Deploy to Rinkeby`) though I don't think they have leveraged that as it is a bit new. Given our code would actually fail on random node version, I think it makes sense to support them with docker for a more stable deployer tool env setup.

Can we add docker support for them? So the ideal situation is this line is run inside a docker with the essential dependency installed already. Or we extend the existing `Deploy to Rinkeby` job with vault deployment.

Reply:
yeah we can def. do that. I'll do that in the next PR so that I don't make this too big.